tailwind.config.js

```diff
@@ -1,7 +1,16 @@
 module.exports = {
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
```

Comment:
good that u found how to extend tailwind config. however, we for the sake of this project cant use tailwind as is, we dont need to extend. the way you extend here is not necessarily a good representation of primary or secondary colors, but that is whole another topic which comes into the topic of design systems.

Reply:
Okay, should I make changes there?

Reply:
sorry, i realized I made a typo in my comment. we CAN use tailwind as is, i mean the default colors in tailwind which is very broad already.https://tailwindcss.com/docs/customizing-colors
We do not need to customize tailwind colors. so u dont need a new pink etc. of course it can be done and it has value, but not in our exercise.

you can simply do bg-pink-500, text-red-200 etc. no need to EXTEND. we can delete that part from config and use default colors in the app.
if you installed tailwind vscode extension, it will suggest you those.